src/Features/Core/Portable/Completion/ArgumentContext.cs

```diff
@@ -29,16 +32,38 @@ internal sealed class ArgumentContext
 
         internal ArgumentProvider Provider { get; }
 
+        /// <summary>
+        /// Gets the document where argument completion is requested.
```

Comment:
might be valuable to indicate this is the original doc, stays constant and is unaffected by further edits while argument-completion is active.

Reply:
This is the document state at the time an argument value is requested.

Reply:
➡️ Updated value documentation